src/main/java/core/basesyntax/serviceimpl/FileWriterImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.service.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
+
+    @Override
+    public void writeToFile(String fileName, String data) {

```

Comment:
According to the checklist, avoid hardcoding file paths. It's good practice to have the file paths passed as parameters to the method, which you are doing here. Ensure that the `fileName` passed to `writeToFile` is a relative path and not hardcoded elsewhere in your application.